### ISSUE TITLE
Ripple 'sticks' if using UI Router in the app under certain conditions

### ISSUE THREAD

**Issue by Todd-Werelius** [angular/material#5434]
This is an odd one, but since so many people use UI-Router it might be worth looking at 

Possibly related to or interacts with https://github.com/angular/material/issues/5294 https://github.com/angular/material/issues/5046

To Reproduce ( Using Chrome + Windows 10 or [Codpen Example](http://codepen.io/Mopholo/pen/LprEdW?editors=001) )

1. Click on a component that has a route attached using ui-sref 
2. Click outside the app ( another tab in browser or whatever)
3. Click back inside the app, notice the ripple is now 'stuck' until you click something inside the app that can get focus

This gif loop probably explains it better ;-) 

![focus-problem](https://cloud.githubusercontent.com/assets/5982829/10826947/60935528-7e29-11e5-808a-78db7092fe74.gif)

**Comment by fassetar** [comment]
+1 seeing it on windows 7 with chrome as well. Looks like md-focused in being added on the return.

**Comment by EladBezalel** [comment]
@DevVersion this might be close after you fix (#5589) will be merged am i right?

**Comment by DevVersion** [comment]
Yes, your right, thanks for referencing that issue.

**Comment by JaySardhara** [comment]
@Todd-Werelius, thanks for including GIF and raising issue. 

@DevVersion facing similar issue for "md-radio-button" with "md-no-ink" flag set. I am getting ripple effect when tab loses focus and gains back again. Also this ripple effect stops after toggling radio buttons few times.
Not raising a new issue as this issue is pretty similar.

Thanks.

**Comment by DevVersion** [comment]
@JaySardhara Yeah can see that. Once #7965 is ready, we should be able to fix this easily. Thanks for the ping

**Comment by JaySardhara** [comment]
@DevVersion that is great !!

**Comment by ThomasBurleson** [comment]
FYI - merge of #7965 has been delayed until 1.1.2 or later.
